pkg/core/object/object.go

```diff
@@ -26,3 +27,20 @@ func AddressOf(obj *object.Object) oid.Address {
 
 	return addr
 }
+
+// ErrNoExpiration means no expiration was set.
+var ErrNoExpiration = errors.New("missing expiration epoch attribute")
+
+// Expiration searches for expiration attribute in the object. Returns
+// ErrNoExpiration if not found.
+func Expiration(obj object.Object) (uint64, error) {
```

Comment:
Do we need this if we're not touching replicator?

Reply:
this can be useful in general as i mentioned in the commit, i just had not searched if more places can reused it